components/Slider/src/MDCSlider.h

```diff
@@ -162,18 +162,34 @@ IB_DESIGNABLE
 - (nullable UIColor *)backgroundTrackTickColorForState:(UIControlState)state;
 
 /**
- The color of the Ink ripple.
+ Sets the thumb radius to use for the specified state.
 
- Defaults to transparent blue.
+ In general, if a property is not specified for a state, the default is to use the @c
+ UIControlStateNormal value. If the @c UIControlStateNormal value is not set, then the property
+ defaults to a default value. Therefore, at a minimum, you should set the value for the normal
+ state.
+
+ @param thumbRadius The radius of the thumb.
+ @param state The state of the slider.
+ @note If a negative value is passed the absolute value will be used.
  */
-@property(nonatomic, strong, nullable) UIColor *inkColor;
+- (void)setThumbRadius:(double)thumbRadius forState:(UIControlState)state;
```

Comment:
Any reason why this cannot be `CGFloat` at the API? It would match the legacy API's type and using explicit `double` types in UI APIs is pretty unusual on iOS.